libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -2,6 +2,7 @@
 #include "AC_PosControl.h"
 #include <AP_Math/AP_Math.h>
 #include <AP_Logger/AP_Logger.h>
+#include <AP_Common/Location.h>
```

Comment:
Why?

Reply:
yep this one as nothing to do here ... I should have paste in the wrong file ... I will double check the other include